source/JsMaterialX/JsMaterialXView/src/index.js

```diff
@@ -202,7 +247,7 @@ function init() {
 
               u_envMatrix: {value: new THREE.Matrix4().makeRotationY(Math.PI)},
               u_envRadiance: {value: radianceTexture},
-              u_envRadianceMips: {value: 1},
+              u_envRadianceMips: {value: Math.log2(Math.max(radianceTexture.image.width, radianceTexture.image.height)) + 1},
```

Comment:
Doesnt this need something like Math.floor or Math.ceil. I thought it was an int